app/assets/javascripts/controllers/dialog_editor/dialog_editor_controller.js

```diff
@@ -65,7 +65,7 @@ ManageIQ.angular.app.controller('dialogEditorController', ['$window', 'miqServic
     translateResponderNamesToIds(dialog.content[0]);
 
     if (requestDialogAction() === 'copy') {
-      dialog.label = dialog.content[0].label = "Copy of " + dialog.label;
+      dialog.label = dialog.content[0].label = 'Copy of ' + dialog.label;
```

Comment:
gettext missing here

Reply:
@romanblanco i am not sure if i should be fixing this in this PR. This is supposed to add and fix existing eslint issues in our code base. I would probably leave this on some other PR. What do you think.

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/4883